src/test/suite/fixtures/recorded/languages/clojure/clearList.yml

```diff
@@ -0,0 +1,23 @@
+languageId: clojure
```

Comment:
Note how quoted lists are treated as lists, but unquoted are not; they're treated as function calls